clippy_lints/src/types.rs

```diff
@@ -1233,6 +1234,7 @@ declare_clippy_lint! {
     /// **Example:**
     /// ```rust
     /// let _ = 2i32 as i32;
+    /// let _ = 0.5 as f32;
```

Comment:
nit (and I see it was like this before): could you add an example of how it should be done? I think it helps to clarify what the lint does

Reply:
Done